Add Vectorized Cloud quick start guide to the docs (#2548)
vectorizedio/redpanda: docs/www/quick-start-cloud-byoc.md

---
title: Vectorized Cloud Quick Start Guide - BYOC
order: 0
---

# Vectorized Cloud Quick Start Guide - BYOC

Want a managed-cloud solution for data streaming that you can host in your own cloud infrastructure?

Vectorized Cloud BYOC (Bring Your Own Cloud) gives you Redpanda services in your cloud that our dedicated team of engineers monitor and manage for you.
You get to handle the cloud contracts and data security, and we make sure you get the best data streaming solution available.

> **_Note_**: If you want Vectorized to manage the cloud infrastructure as well, [create a fully-managed cluster](/docs/quick-start-cloud-managed).

After you set up the cluster, you'll get an endpoint and security credentials to allow your clients to access the cluster.

> You can use PandaProxy HTTP requests to manage your topics and partitions,

Author: ```suggestion
You can use PandaProxy HTTP requests to produce and consume on the cluster,
```

but we also give you a graphical dashboard with real-time metrics, basic topic management, and security controls for cluster and data access.

## Sign in to Vectorized Cloud

> You can register for an account at [cloud.vectorized.io](cloud.vectorized.io) with an email address and password or with a Google account.

Author: ```suggestion
You can register for an account at [vectorized.cloud](vectorized.cloud) with an email address and password or with a Google account.
```

## Add a namespace

Vectorized Cloud clusters are grouped into namespaces to help you organize your clusters in groups for convenience,
but cluster names have to be unique across the whole organization.

Click **Add namespace** to create a namespace and give it a name.

## Add a cluster

In your namespace, click **Add cluster** and then choose which type of cluster to add:

First, you create the cluster in the Vectorized Cloud UI.

1. Enter a name for the cluster.
2. Select a cloud provider: **AWS** or **GCP**

> 3. Select a region in the provider.

Author: Add storage types

> After the cluster is created in Vectorized Cloud, click **Setup cluster security** and follow the instructions to create the cluster in your cloud account.

Author: Update for new setup instructions

4. Connect to your cloud project.
5. Download and apply the Kubernetes manifest to deploy the cluster.

## Create a service account

Before you can connect to the cluster you have to create a service account for user authentication.

To create a service account with full permissions on the cluster:

1. Go to the **Security** section of the cluster.
2. In **Service accounts**, click **Add service account**.
3. Name the service account and click **Create**.
    The password for the account is automatically generated for the account.
    Make sure that you copy the password and keep it safe.
    You cannot show the password again or change it.
4. In the **ACL** section, click **Add ACL**.
5. Select the service account and permissions that you want it to have.
    To give the account permissions on all topics and groups, enter `*` for the topic and groups IDs.
6. Click **Create**.

Now you can use the service account in your cluster connections.

## Get the connection resources for the cluster

When the cluster is in **Running** state, click on the cluster to see the cluster management actions.

You can find the connection details for your cluster in the **Topics** section.

> - Cluster host addresses - Clients connect to these addresses for produce and consume actions.

Author: ```suggestion
- Cluster host addresses - [Clients](/docs/faq/) use these addresses to connect to the Kafka API.
```

> - Pandaproxy host - You can use [REST API calls](https://vectorized.io/blog/pandaproxy/) to manage your cluster.

Author: ```suggestion
- Pandaproxy host - You can use [REST API calls](https://vectorized.io/blog/pandaproxy/) to produce and consume on the cluster.
```

Connections to the cluster require TLS authentication with the TLS certificates in the **Security** section.

> - Kafka API TLS - Download these certificates for `rpk` cluster management and client data connections.

Author: ```suggestion
- Kafka API TLS - Download these certificates for `rpk` cluster management and Kafka client connections.
```

> - Proxy API TLS - Download these certificates for REST API management connections.

Author: ```suggestion
- PandaProxy API TLS - Download these certificates for REST API connections.
```
